gammapy/data/observations.py

```diff
@@ -635,6 +635,27 @@ def select_time(self, time_intervals):
     def _ipython_key_completions_(self):
         return self.ids
 
+    def split(self, groups_id):
+        """Split obsevations in multiple groups of observations
+
+        Parameters
+        ----------
+        cluster_id : array
+            Array of cluster ID
+
+        Returns
+        -------
+        obs_clusters : list of `~gammapy.data.Observations`
```

Comment:
Minor: you could consider returning a `dict` here as well, which keeps a clearer association between the group id and the group.